src/components/ActiveTodos/ActiveTodos.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type Props = {
+  activeItems: number;
+};
+
+export const ActiveTodos: React.FC<Props> = ({ activeItems }) => {
+  return (
+    <span className="todo-count" data-cy="todosCounter">
+      {`${activeItems} items left`}
```

Comment:
you had a lot of code for this `activeItems` state. In my opinion, it would be better to find a length of filtered array by completed status.